internal/logging/zerolog.go

```diff
@@ -57,13 +56,22 @@ func decorate(l zerolog.Logger) zerolog.Logger {
 // InitializeStdout initializes logging to standard output with human-friendly output.
 // It is used before CloudWatch logging output is initialized, or in unit and integration tests.
 func InitializeStdout() {
+	var noColor bool
+
+	// Do not use colors when running in clowder
+	if config.InClowder() {
```

Comment:
I know that noColor is false by default, but we are calling the config.InClowder() anyways.
Could we (1) possibly just initialize the noColor with the `config.InClowder()`? The return value is always bool anyways... Or do you see a problem with readability? I do not want this to block the PR, and I know it is not a problem in Go.

Reply:
Haha yeah of course, this happens when you end up refactoring over and over again until you left with such a non-sense. Good point.